01/app.js

```diff
@@ -0,0 +1,21 @@
+const formEl = document.querySelector("form");
+const firstNameEl = document.querySelector('input[name="firstName"]');
+const lastNameEl = document.querySelector('input[name="lastName"]');
+const ulEl = document.querySelector("ul");
+
+formEl.addEventListener("submit", handleSubmit);
+
+function handleSubmit(e) {
+	e.preventDefault();
+	const firstNameValue = firstNameEl.value;
+	const lastNameValue = lastNameEl.value;
+
+	if ((firstNameValue !== "") & (lastNameValue !== "")) {
```

Comment:
Uwaga na operator! Powinno być `&&`. 
Jeden ampersand to co innego: https://stackoverflow.com/questions/7310109/whats-the-difference-between-and-in-javascript